test/cql-pytest/test_wasm.py

```diff
@@ -1107,3 +1107,13 @@ def test_counter(cql, test_keyspace, scylla_only):
             cql.execute(f"UPDATE {table} SET c = c + 1  WHERE p = 42;")
             cql.execute(f"UPDATE {table} SET c = c - 4  WHERE p = 42;")
             assert cql.execute(f"SELECT {ri_counter_name}(c) AS result FROM {table} WHERE p = 42").one().result == -1
+
+# Test that we can use a wasm binary converted to hex as the source of a WASM UDF.
+def test_wasm_base64(cql, test_keyspace, table1, scylla_only):
+    mul_name = 'mul'
+    mul_source = open(os.path.realpath(os.path.join(__file__, f"../../resource/wasm/mul.b64")), 'r').read()
```

Comment:
I'm curious, why is mul.b64 so long? Could we have a much much shorter file? If we could, we could have it inline instead of requiring an external file, which although acceptable, isn't really the nicest thing (it makes the test have weird external dependencies that aren't immediately obvious).

Reply:
Another nitpick: I don't think the realpath call is necessary in the above line.

Reply:
Hmm, I change my mind on the external file if this external file is automatically generated by the rust compiler. Then it is fine.

Reply:
The external file is a result of calling `base64 -w0` on the file generated by the compiler. I'm removing the realpath in a rebase.

Reply:
Actually, looks like the realpath (or something similar) is needed here - without it I'm getting `NotADirectoryError` exceptions